RELEASENOTES.md

```diff
@@ -0,0 +1,4545 @@
+# Release notes {#release_notes}
+
+Release notes are prepared by Artefactual for each release of
+Archivematica. Most releases also have a corresponding release of the
+Storage System, and may also include corresponding releases for various
+dependencies and libraries (i.e. FIDO, Automation Tools) - be sure to
```

Comment:
Instead of FIDO as an example (which we do not maintain) I would change to AIPScan, for e.g.